<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>750</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Directory">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>610</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sel_Folder">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Include_SubDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Include Subdirectries</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>960</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Subdirectories found:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SubDir_Found">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pictures found:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pic_Found">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Compare_SubDir">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare All Pictures</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>220</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>960</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="picture_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="picture_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dimensions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dimensions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dimension_Pic_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 x 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dimension_Pic_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 x 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>640</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_Pic_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 MB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size_Pic_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>680</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 MB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name_Pic_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>foo.jpg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name_Pic_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>bar.jpg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Keep_Both">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- Keep Both -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Keep_Left">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- Keep Left Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Keep_Right">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep Right Picture -&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Match:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Image_Match">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
